ecs-cli/modules/cli/compose/project/project.go

```diff
@@ -51,6 +52,9 @@ type Project interface {
 type ecsProject struct {
 	project.Project
 
+	// TODO: use this instead of project.Project
+	containerproject.ContainerProject
```

Comment:
As per our discussion IRL, I think that embedding ContainerProject is unnecessary, and that the API on this object can be moved into ecsProject directly.